dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnector.java

```diff
@@ -380,6 +381,85 @@ protected String toCallDescription() {
     }
   }
 
+  private static class PartitionsJsonlTask extends AbstractHiveMetadataTask {
+
+    private PartitionsJsonlTask(Predicate<String> databasePredicate) {
```

Comment:
The `databasePredicate` here can be annotated `@Nonnull`, to match the `super` call before.
Alternatively, a single `@ParametersAreNonnullByDefault` can be used to replace all `@Nonnull` annotations on parameters for the class.

Reply:
Fixed by adding `@ParametersAreNonnullByDefault`.